doc-content/drools-docs/src/main/asciidoc/DMN/dmn-execution-remote-java-proc.adoc

```diff
@@ -11,13 +16,22 @@ endif::[]
 ifdef::DROOLS,JBPM,OP[]
 <<_installationandsetup>>.
 endif::[]
-* A KIE container is deployed in {KIE_SERVER} in the form of a KJAR that includes the DMN model. For information about project packaging and deployment, see
+* A KIE container is deployed in {KIE_SERVER} in the form of a KJAR that includes the DMN model, ideally compiled as an executable model for more efficient execution:
++
+--
+[source]
+----
+mvn clean install -DgenerateDMNModel=yes
+----
+
+For information about project packaging and deployment and executable models, see
```

Comment:
Add "more" as per IBMSG? and add a comma?

s/For information about project packaging and deployment and executable models, see/For more information about project packaging and deployment, and executable models, see